packages/service/ni_measurement_plugin_sdk_service/measurement/client_support.py

```diff
@@ -13,3 +24,89 @@
     "ParameterMetadata",
     "serialize_parameters",
 ]
+
+
+def deserialize_parameters(
+    parameter_metadata_dict: Dict[int, ParameterMetadata],
+    parameter_bytes: bytes,
+    message_name: str,
+    *,
+    convert_paths: bool = True
+) -> Sequence[Any]:
+    """Deserialize parameter bytes into separate parameter values.
+
+    Args:
+        parameter_metadata_dict: Parameter metadata by ID.
+
+        parameter_byte: Byte string to deserialize.
+
+        message_name: gRPC message name (e.g. f"{service_class}.Outputs").
+
+        convert_paths: Specifies whether to convert path parameters to pathlib.Path.
+
+    Returns:
+        Deserialized parameter values, ordered by ID.
+    """
+    parameter_values = _internal_deserialize_parameters(
+        parameter_metadata_dict, parameter_bytes, message_name
+    )
+
+    for id in parameter_values.keys():
+        metadata = parameter_metadata_dict[id]
+        if (
+            convert_paths
+            and metadata.type == FieldDescriptorProto.TYPE_STRING
+            and metadata.annotations
+            and metadata.annotations.get("ni/type_specialization") == "path"
```

Comment:
Likewise, here as well, can we use the TYPE_SPECIALIZATION_KEY constant from [_annotations.py](https://github.com/ni/measurement-plugin-python/blob/801220f17686e1e1b9acf27444078b2e15f69361/packages/service/ni_measurement_plugin_sdk_service/_annotations.py#L4) here?